ci-operator/step-registry/hypershift/mce/ibmz/create/hcp/hypershift-mce-ibmz-create-hcp-ref.yaml

```diff
@@ -20,6 +20,9 @@ ref:
       documentation: "Hosted Cluster namespace"
     - name: AGENT_IBMZ_CREDENTIALS
       default: /etc/hypershift-agent-ibmz-credentials
+    - name: ENABLE_ICSP
```

Comment:
@heliubj18 , I think it would be better to add the flag name ENABLE_MGMT_ICSP for better understanding and documentation with more relevancy